test/xebow_test.exs

```diff
@@ -17,8 +17,16 @@ defmodule XebowTest do
     end
   end
 
-  test "has layout" do
-    assert %Layout{} = Xebow.layout()
+  test "has a layout for the keybow" do
+    assert %Layout{} = Xebow.layout(:keybow)
+  end
+
+  test "has a layout for excalibur" do
+    assert %Layout{} = Xebow.layout(:excalibur)
+  end
```

Comment:
:information_source: (optional)

I feel like these tests are more-so formalizing an architectural decision that we admittedly don't really like, rather than testing functionality we want to avoid breaking.

Reply:
yes, I was just extrapolating the pattern